lib/Web/Server.hs

```diff
@@ -59,7 +147,7 @@ reportErrors io = do
       throwError err500 { errBody = errorToServantErrorMsg ioError }
```

Comment:
By the way, in `errorToServantErrorMsg`, we should use `displayException` instead of `show`. It usually displays a more human-readable version of the exception.

Reply:
Also, instead of going
```
String -> Strict.Text -> Strict.ByteString -> Lazy.ByteString
```

We can go
```
String -> Lazy.Text -> Lazy.ByteString
```

Using `Data.Text.Lazy.pack` and `Data.Text.Lazy.Encoding.encodeUtf8`

Reply:
Done